app/controllers/cookies_preferences_controller.rb

```diff
@@ -10,7 +10,10 @@ def create
 
     if @cookies_preferences_form.valid?
       cookies["consented-to-cookies"] = { value: @cookies_preferences_form.cookies_consent, expires: 6.months.from_now }
-      redirect_to session[:previous_url]
```

Comment:
this could just be
`redirect_to session[:previous_url] unless params[:no_redirect]`